SQL/SQL.sublime-syntax

```diff
@@ -249,23 +271,59 @@ contexts:
       # Handle situations where the schema and . 
     - match: '{{end_identifier}}'
       pop: true
-  like_escape:
-    - match: (\')([^'])(\')
-      captures:
-        0: meta.string.escape.sql string.quoted.single.sql
-        1: punctuation.definition.string.begin.sql
-        2: constant.character.escape.sql
-        3: punctuation.definition.string.end.sql
+  create-condition:
```

Comment:
Do we want to rename all context so they use dashes any time soon? Maybe this and adding an empty line between each, to make the syntax definition look more modern.

Doesn't need to happen in this PR, but maybe in a "maintanance follow-up"?

Reply:
sounds good to me :)

Reply:
I renamed the contexts as part of this PR, most of them were introduced here anyway ;)